addons/website/static/tests/tours/create_website.js

```diff
@@ -0,0 +1,33 @@
+
+odoo.define('test_website_modules.tour.test_create_website', function (require) {
+    'use strict';
+
+    const tour = require('web_tour.tour');
+
+    tour.register('test_create_website', {
```

Comment:
We never indent the first level of code inside `odoo.define`, to gain an indent level, as we would systematically lose a level otherwise

Note that with the new JS way of writing it, there is no such thing, which is also a good reason to save that level indent -> When migrated to the new JS, we won't have to unindent the code and have a diff on the whole file